analysis/src/Loc.ml

```diff
@@ -8,3 +8,5 @@ let toString (loc : t) =
   (if loc.loc_ghost then "__ghost__" else "") ^ (loc |> range |> Range.toString)
 
 let hasPos ~pos loc = start loc <= pos && pos < end_ loc
+
+let hasPosInclusive ~pos loc = start loc <= pos && pos <= end_ loc
```

Comment:
Yeah, I really don't know... 😄 I understand this shouldn't be needed, but it was the only way I found to make it work. I'll try again and see if we can maybe figure out how to avoid this together.